prow/config.yaml

```diff
@@ -314,6 +314,11 @@ presets:
     env:
       - name: BUILD_TYPE
         value: master
+  - labels:
+      preset-build-busola-master: "true"
+    env:
+      - name: BUILD_TYPE
+        value: master
```

Comment:
`BUILS_TYPE` is not supported by `build-generic.sh` script. The parameter will be ignored.

Reply:
Also this preset doesn't seem to be used anywhere...